modules/products/jira/main.tf

```diff
@@ -28,3 +42,41 @@ module "database" {
   db_master_username      = var.db_master_username
   db_master_password      = var.db_master_password
 }
+
+resource "kubernetes_persistent_volume" "share-home-pv" {
```

Comment:
We could potentially move PV and PVC to NFS module and expose PVC name

Reply:
That's a good idea, I already done this refactor partially before. Actually I created a utility module to get the nfs server ip and create a PV and PVC with requested size. I can complete it and raise a PR.

Reply:
I think we should get @nghazali PR merged in first and then modify this PR to use it.